python/tests/test_pyvw.py

```diff
@@ -182,4 +193,94 @@ def test_parse():
 
     ex = model.parse(["| a:1 b:0.5", "0:0.1:0.75 | a:0.5 b:1 c:2"])
     assert len(ex) == 2
+
+    # To test the prediction on multiline examples
+    finish = model.finish_example(ex)
+    assert model.predict(ex) == [0.0, 0.0]
     del model
+
+def test_numspace_id():
+    vw_ex = vw(quiet=True)
+    ex = vw_ex.example('1 |a two features |b more features here')
+    nm1 = pyvw.namespace_id(ex, 2)
+    assert nm1.id == 2
+    assert nm1.ord_ns == 128
+    assert nm1.ns == '\x80' # Represents string of ord_ns
+    nm2 = pyvw.namespace_id(ex, 'my_example')
```

Comment:
This does not make sense. Granted it is an unclear part of the API. You are invoking the branch which does not touch the ex object and so doesn't really make sense for this test case?

Reply:
Okay, for now, I will keep this for later when I would go for left out ones.